packages/charting/src/axes.jsx

```diff
@@ -182,13 +183,13 @@ export class TickComponent extends React.Component {
             barWidth={barWidth}
             rotate={rotate}
             correctness={correctness}
-            error={error}
+            error={error && (error[index] || error[index] == '')}
```

Comment:
Do we need `error[index] == ''`?

Reply:
no, thank you, we do not need it anymore. I will remove it.